cppForSwig/Signer/ResolverFeed_Wallets.cpp

```diff
@@ -20,7 +20,7 @@ using namespace Armory::Wallets;
 //// ResolverFeed_AssetWalletSingle
 ////
 ////////////////////////////////////////////////////////////////////////////////
-void ResolverFeed_AssetWalletSingle::addToMap(shared_ptr<AddressEntry> addrPtr)
+void Armory::Signer::ResolverFeed_AssetWalletSingle::addToMap(shared_ptr<AddressEntry> addrPtr)
```

Comment:
This change is way too disruptive, the cpp file should use namesapce Armory::Signer as a one liner alternative

Reply:
See response for Transactions.cpp - for some reason full namespace is required to build these classes. Otherwise it's a compile error (not warning).